components/MultiSelect.tsx

```diff
@@ -10,18 +10,23 @@ interface Option {
 interface MultiSelectProps {
   name: string;
   options: Option[];
-  placeholder: string;
+  // eslint-disable-next-line react/require-default-props
```

Comment:
Should we add this `eslint` exception globally?

Reply:
Same question, saw it at a couple of other places as well.

Reply:
Thanks, good catch -- disabled globally now, I think we can remove it once we stop using `prop-types`